packages/composites/anchor/test/unit/price.test.ts

```diff
@@ -1,6 +1,6 @@
 import { Requester } from '@chainlink/ea-bootstrap'
 import { assertError } from '@chainlink/ea-test-helpers'
-import { AdapterRequest } from '@chainlink/types'
+import { AdapterRequest } from '@chainlink/ea-bootstrap'
```

Comment:
I wonder if there's an easy way to combine multiple `'@chainlink/ea-bootstrap'` imports. Not worth it to do manually, but maybe it wouldn't be too bad to automate something?

Reply:
Done!